README.md

```diff
@@ -2,7 +2,7 @@
 
 <a href="https://littlehorse.io/"><img alt="littlehorse.io" src="https://raw.githubusercontent.com/littlehorse-enterprises/littlehorse/01ef9618b7b719c3aac1d2e2eb5096f164a659b1/docs/images/badge-littlehorse-io.svg"/></a>
 <a href="https://littlehorse.dev/"><img alt="littlehorse.dev" src="https://raw.githubusercontent.com/littlehorse-enterprises/littlehorse/01ef9618b7b719c3aac1d2e2eb5096f164a659b1/docs/images/badge-littlehorse-dev.svg"/></a>
-[![slack](https://img.shields.io/badge/Slack-511651.svg?logo=slack)](https://kubernetes.slack.com/messages/C87MF1RFD)
+[![slack](https://img.shields.io/badge/Slack-511651.svg?logo=slack)](https://launchpass.com/littlehorsecommunity)
```

Comment:
No idea where this came from, but it definitely didn't point to our slack. We aren't on `kubernetes.slack.com`

Reply:
Great catch, that was probably a typo on my end from adding the Shields but I have no idea where I got a link to the Kubernetes slack.